### PR TITLE
fix(MobilityDataValidation): slow on large gtfs files

### LINKED ISSUE #524: MobilityData Validator CSV Parsing Issue with large GTFS files
## Observed behavior

MobilityData Validation Result can run into CSV parsing issues when validating the MBTA's GTFS feed. Running the same feed with the MobilityData Validator locally does not produce the same result.  

## Expected behavior

Validator should not produce the parsing issue. 

## Steps to reproduce the problem

Validate MBTA's latest GTFS feed. Observe the following errors: 
image

## Any special notes on configuration used

None 

## Version of datatools-server and datatools-ui if applicable (exact commit hash or branch name)

Latest validator branch, dev server.

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

The MobilityData validator takes a while between finishing and saving results. There is no way to check when this process completes using their api. The solution is to do a wait, currently 5 seconds. However, on large feeds this is not enough. The solution is to adjust the wait depending on file size.

Closes #524 

Looking to try this fix on as wide a variety of systems as possible to make sure the values are correct

### CODE REVIEW

**Review by philip-cline — APPROVED** [review]

File: src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java [hunk 1]
```diff
@@ -393,8 +393,8 @@ public void validateMobility(MonitorableJob.Status status) {
             status.update("MobilityData Analysis...", 11);
 
             // Wait for the file to be entirely copied into the directory.
-            // TODO: base this on the file being completely saved rather than a fixed amount of time.
-            Thread.sleep(5000);
+            // 5 seconds + ~1 second per 10mb
+            Thread.sleep(5000 + (this.fileSize / 10000));
```
Comment (binh-dam-ibigroup) [inline src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java:397]: What unit is `this.fileSize`? According to the code, it is set with `[newGtfs]File.length()` which is in bytes.

Comment (miles-grant-ibigroup) [reply src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java:397]: I'm not sure, but this does fix things

Comment (binh-dam-ibigroup) [reply src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java:397]: @miles-grant-ibigroup Can you fix the comment so it corresponds to the code?

Comment (miles-grant-ibigroup) [reply src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java:397]: Did I get the decimal math right in 0ebf2227078653ede1d54a77676b22bb4a89a726?

Comment (binh-dam-ibigroup) [reply src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java:397]: I'm sorry, the initial comment was correct, juggling with the units was confusing (10,000,000 / 10,000 = 1000).

**Review by binh-dam-ibigroup — APPROVED** [review]
It was indeed 1 sec / 10 MB...

**Comment by miles-grant-ibigroup** [comment]
Not sure why it says closed instead of merged...

**Pull request closed without merging**